src/importexport/imagesexport/iimagesexportconfiguration.h

```diff
@@ -33,13 +33,16 @@ class IImagesExportConfiguration : MODULE_EXPORT_INTERFACE
 
     // Pdf
     virtual int exportPdfDpiResolution() const = 0;
+    virtual void setExportPdfDpiResolution(const int dpi) const = 0;
```

Comment:
We already use std::optional<float> type for dpi in setExportPngDpiResolution. I think you should use this type here also